pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -2840,6 +3044,102 @@ The requested number of points must be not larger than 65336.`,
 			tree.VolatilityImmutable,
 		),
 	),
+	"st_zmin": makeBuiltin(
+		defProps(),
+		geometryOverload1(
+			func(ctx *tree.EvalContext, g *tree.DGeometry) (tree.Datum, error) {
+				t, err := g.Geometry.AsGeomT()
```

Comment:
you should be able to infer the geometry bounding box from `g.Geometry.BoundingBox`. if it is nil, it is empty.

Reply:
done